dom.bs

```diff
@@ -4175,57 +4157,80 @@ returns as mask:
  <li><dfn const for=Node>DOCUMENT_POSITION_IMPLEMENTATION_SPECIFIC</dfn> (32, 20 in hexadecimal).
 </ul>
 
-The <dfn method for=Node>compareDocumentPosition(<var>other</var>)</dfn>
-method must run these steps:
+<p>The <dfn method for=Node><code>compareDocumentPosition(<var>other</var>)</code></dfn> method,
+when invoked, must run these steps:
 
 <ol>
- <li>Let <var>reference</var> be the <a>context object</a>.
- <li>If <var>other</var> and <var>reference</var> are the
- same object, return zero.
+ <li><p>If <a>context object</a> is <var>other</var>, then return zero.
+
+ <li><p>Let <var>node1</var> be <var>other</var> and <var>node2</var> be <a>context object</a>.
+
+ <li><p>Let <var>attr1</var> be null.
```

Comment:
Wow, this algorithm was more complicated than I thought. Is this a reverse engineering of Gecko's algorithm? I think I can spot some differences to what Blink does, and am not sure if they matter.